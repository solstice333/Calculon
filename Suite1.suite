P a.out main.c header.h
T test1.in test1.out 1000 -a -b -c
T test2.in test2.out 1000 -xyz
T test3.in test3.out 800
P b.out bprogram.c SmartAlloc.c SmartAlloc.h
T test5.in test5.out 10000
T test4.in test4.out 30000 -verbose
