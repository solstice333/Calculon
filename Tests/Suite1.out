a.out 3 of 3 tests passed.
b.out 2 of 2 tests passed.
